lib/datasource/bitbucket-tags/index.ts

```diff
@@ -23,12 +23,16 @@ export class BitBucketTagsDatasource extends Datasource {
     super(BitBucketTagsDatasource.id);
   }
 
-  static getRegistryURL(registryUrl: string): string {
+  static getRegistryURL(registryUrl?: string): string {
     // fallback to default API endpoint if custom not provided
     return registryUrl ?? this.defaultRegistryUrls[0];
   }
 
-  static getCacheKey(registryUrl: string, repo: string, type: string): string {
+  static getCacheKey(
+    registryUrl = this.defaultRegistryUrls[0],
```

Comment:
```suggestion
    registryUrl: string | undefined,
```
This is already done in `getRegistryURL`